### ISSUE TITLE
v5 - ERR_REQUIRE_ESM error

### ISSUE THREAD

**Issue by apexskier** [mike-marcacci/node-redlock#98]
On Node 12 and 14, given the following files:

`index.ts`
```ts
import Redlock from "redlock";
import IORedis from "ioredis";

const connection = new IORedis();
const redlock = new Redlock([connection]);
```

`package.json`
```json
{
  "dependencies": {
    "ioredis": "^4.27.7",
    "redlock": "^5.0.0-alpha.0"
  },
  "devDependencies": {
    "typescript": "^4.3.5"
  }
}
```

Run `yarn` and `./node_modules/.bin/tsc index.ts`. You'll get compilation errors (trimmed for brevity).

```
node_modules/redlock/dist/index.d.ts:1:23 - error TS2688: Cannot find type definition file for 'node'.

1 /// <reference types="node" />
                        ~~~~
```

Those are resolvable by including node typings (`yarn add -D @types/node`).

Next, after building, run the program with `node index.js` and you'll get this error:

```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/cameronlittle/Developer/test/node_modules/redlock/dist/index.js
require() of ES modules is not supported.
require() of /Users/cameronlittle/Developer/test/node_modules/redlock/dist/index.js from /Users/cameronlittle/Developer/test/index.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename /Users/cameronlittle/Developer/test/node_modules/redlock/dist/index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /Users/cameronlittle/Developer/test/node_modules/redlock/package.json.
```

**Comment by apexskier** [comment]
It looks like this also reproduces in the node repl - require("redlock") throws the same error.

**Comment by apexskier** [comment]
I think the issue is that the distributed code uses `export` syntax. 

From the node repl, this works
```js
import("redlock").then(redlock => console.log(redlock))
```

**Comment by mike-marcacci** [comment]
@apexskier yes you are correct here. I am going to be re-publishing this in both es modules and commonjs, which should make this more ergonomic in commonjs contexts.

**Comment by mike-marcacci** [comment]
I just published `v5.0.0-beta.1` which should fix this.
